### PR TITLE
Nemeth tests

### PR DESCRIPTION
**Pull request by NSoiffer**
Added an initial 12 tests for MathML -> Nemeth.
5 pass
7 fail
Issues will be added shortly.

### CODE REVIEW

**Comment by NSoiffer** [comment]
Updated Makefile.am to add XFAIL for failing tests

**Comment by bertfrees** [comment]
Great. Let me know if you want me to merge this now or whether you first want to add more to the branch.

**Comment by NSoiffer** [comment]
Merge it now. I'll be adding more tests over the next week, but there is no reason to wait to get the initial tests along with some fixes to the tables to make them compatible with the current liblouis tables.

File: tests/mathml_nemeth/general_001.test/Makefile.am [hunk 1]
```diff
@@ -0,0 +1,3 @@
+EXTRA_DIST =		\
+	expected.txt	\
+	input.xml
```
Comment (egli) [inline tests/mathml_nemeth/general_001.test/Makefile.am]: If you want the png to be in the distribution tarball you should mention it here in `EXTRA_DIST`.

File: tests/mathml_nemeth/mfrac_01.test/Makefile.am [hunk 2]
```diff
@@ -0,0 +1,3 @@
+EXTRA_DIST =		\
+	expected.txt	\
+	input.xml
```
Comment (egli) [inline tests/mathml_nemeth/mfrac_01.test/Makefile.am]: Again you might want to add the png

**Comment by bertfrees** [comment]
@NSoiffer OK.

@egli I'll do the merge.

**Comment by NSoiffer** [comment]
I've added the images to all the test Makefile.am files. Also more tests were added. The majority of them fail though :-(.

Should be good to go for a merge.

**Comment by NSoiffer** [comment]
I've added more tests. There are 82 tests now, 68 of which fail. I'll eventually add a character sweep test, but for now, I plan to work on fixing some issues using xpath as per a discussion with Bert. I'll do that on a different branch.

**Pull request merged**